### DISCUSSION TITLE
Some of data not shown from CSV

### DISCUSSION THREAD

**Discussion by wunj2321** [Budibase/budibase#11283] [Help] [answered]
When i upload file some of the data shown something like this, so i don't sure what happen to it. ( K and M column) 
![messageImage_1689493078646](https://github.com/Budibase/budibase/assets/140029788/b3ab5465-44a1-49b6-a019-9ac7372b7305)

[Chem.csv](https://github.com/Budibase/budibase/files/12101288/Chem.csv)
[Micro.csv](https://github.com/Budibase/budibase/files/12101289/Micro.csv)

**Comment by wunj2321** [comment]
[Object object] and i have give the csv below

**Reply by ConorWebb96** [reply, marked as answer]
Hey @wunj2321,

You can still technically access this information. All you need to do is drill into that object.

The fix for this would be making the title different as you have named it `E.Coli`, if you removed the `.` and name it `E Coli` this would fix it and output it correctly. 
<img width="69" alt="Screenshot 2023-07-20 at 07 57 09" src="https://github.com/Budibase/budibase/assets/126772285/de9002f4-f26c-437b-b5c1-908fc23b63c4">

When testing this was all I changed and everything imported correctly. I'm not 100% sure on what the cause is, it must be related to how Budibase import works. It seems to take whatever is after the `.` and use that as the key for creating the object.
<img width="36" alt="Screenshot 2023-07-20 at 07 57 16" src="https://github.com/Budibase/budibase/assets/126772285/d80d3844-ad7b-40a6-bfb7-d23faf443aeb">

Heres the output on the frontend before I fixed the title:
<img width="652" alt="Screenshot 2023-07-20 at 08 02 40" src="https://github.com/Budibase/budibase/assets/126772285/98461343-5539-4db1-bd4e-cfc6e72f9b15">

And heres what happened after I removed the `.` in the title after importing.
<img width="209" alt="Screenshot 2023-07-20 at 07 57 02" src="https://github.com/Budibase/budibase/assets/126772285/4977b110-882a-4f2f-a1a3-ad0095e0134f">


I hope this makes sense and helps resolve your issue.

**Comment by wunj2321** [comment]
Thank you @ConorWebb96 , and one more question if i upload 2 file in one DB it shown nothing, I would like to know what it is happen? 
![messageImage_1689493012261](https://github.com/Budibase/budibase/assets/140029788/c199189c-ba6c-4212-9406-035f63427767)

Thank you once again

**Reply by ConorWebb96** [reply]
Hey @wunj2321,

I believe this to be related to your `Chem.csv` file. It is very poorly structured you have some rows appearing as column headings when they should be apart of the first row. (The below screenshots all appear on the same line.)
<img width="1010" alt="Screenshot 2023-07-20 at 10 26 44" src="https://github.com/Budibase/budibase/assets/126772285/fce88ca1-c93a-440e-af42-260371bbd3ca">
<img width="889" alt="Screenshot 2023-07-20 at 10 27 12" src="https://github.com/Budibase/budibase/assets/126772285/077ee4dc-448c-4986-8c3b-4fe996ea1881">

You also have stray quotation marks at random places within the CSV file. I would suggest fixing these issues and then re-importing your CSV file.

I'm not 100% sure why it has affected the other table though. I'm currently still investigating this, it likely is a bug I will link it here whenever I've got more information and have written up the bug report.
